### DISCUSSION TITLE
Is there a secret setting to make the style wizard/editor open in a small window?

### DISCUSSION THREAD

**Discussion by krystian3w** [openstyles/stylus#1643] [Q&A] [answered]
I am very often annoyed that it starts the editor or style builder in a large window with an address bar, tabs, bookmarks, that I sometimes use another add-on to switch to the mode, that I only have the window title bar (system).

It would be nice if it could be forced in the add-on without hacking with another add-on.

**Comment by tophf** [comment, marked as answer]
Enable `Open editor in a new window` and `Use a simple window (no omnibox)` in options. In the next version of Stylus the second option will have a different name: `...without address bar`.

**Reply by krystian3w** [reply]
In Polish I got: `Użyj prostego okna (bez omniboksu)`, so in next will be:

`Użyj prostego okna bez paska adresu`.

Or translation is not ready due lack of Poland speakers?

**Reply by tophf** [reply]
It's [already translated](https://explore.transifex.com/github-7/Stylus/) but not yet released.
